seems to have got bronken with this commit:

http://bazaar.launchpad.net/~mysql/mysql-utilities/trunk/revision/148#mysql/utilities/common/server.py

or maybe even earlier

http://bugs.mysql.com/bug.php?id=66863
also http://bugs.mysql.com/bug.php?id=67512

--- mysql-utilities-1.1.0/./mysql/utilities/common/server.py~	2012-09-26 23:50:47.000000000 +0300
+++ mysql-utilities-1.1.0/./mysql/utilities/common/server.py	2012-10-05 01:30:37.440326297 +0300
@@ -330,7 +330,7 @@
     if not _require_version(source, version):
         raise UtilError("The %s version is incompatible. Utility "
                         "requires version %s or higher." %
-                        (source.name, version))
+                        (source.host, version))
 
     # If not cloning, connect to the destination server and check version
     if not cloning:
--- mysql-utilities-1.1.0/./mysql/utilities/common/server.py~	2012-10-05 01:31:09.000000000 +0300
+++ mysql-utilities-1.1.0/./mysql/utilities/common/server.py	2012-10-05 01:42:07.724991092 +0300
@@ -343,7 +343,7 @@
         if not _require_version(destination, version):
             raise UtilError("The %s version is incompatible. Utility "
                             "requires version %s or higher." %
-                            (destination.name, version))
+                            (destination.host, version))
     elif not quiet and dest_dict is not None and \
          not isinstance(dest_val, Server):
         _print_connection(dest_name, src_dict)
